tests/test_viewing.py

```diff
@@ -621,3 +621,44 @@ def test_view_equals_module(SimpleComp, SimpleBranch):
             )
         ]
     )
+
+
```

Comment:
you might also wanna add a line to `test_view_equals_module` to make sure the attr behaves the same for views and modules